macosx/PiecesView.mm

```diff
@@ -151,22 +151,22 @@ - (void)updateView
 
     // get the current state
     BOOL const showAvailability = [NSUserDefaults.standardUserDefaults boolForKey:@"PiecesViewShowAvailability"];
-    NSInteger const numCells = MIN(_torrent.pieceCount, kMaxCells);
+    int const numCells = static_cast<int>(MIN(_torrent.pieceCount, kMaxCells));
     PieceInfo info;
     [self.torrent getAvailability:info.available size:numCells];
     [self.torrent getAmountFinished:info.complete size:numCells];
 
     // compute bounds and color of each cell
-    NSInteger const across = (NSInteger)ceil(sqrt(numCells));
+    int const across = (int)ceil(sqrt(numCells));
```

Comment:
nit: `static_cast<int>(...)`?

Reply:
It's also used in libtransmission. ;)
https://github.com/transmission/transmission/blob/c14094f4c9c05455c58e1e503d0d31001e679401/libtransmission/makemeta.cc#L227C4-L227C4

But fair, I'll fix the one in macOS code.